tests/testTicket2352.py

```diff
@@ -51,15 +54,18 @@ def testExtName(self):
         self.checkExtName("THREE", 3, 4)
 
     def checkExtNum(self, hdu, extNum):
+        if hdu is None:
+            hdu = -(1 << 31)            # == INT_MIN
```

Comment:
This is the only place I had to do it, and chatting with {J,P}im it would be a pain -- the default value appears in many many places in the pybind11ings.  With the advent of named arguments I don't think this is going to be a problem -- if you disagree, please file a ticket.